transport_nantes/mobilito/urls.py

```diff
@@ -16,4 +16,6 @@
          name='session_summary'),
 
     path('ajax/create-event/', views.create_event, name='event_creation'),
+    path('ajax/get-address/', views.MapsGeocodingView.as_view(),
+         name='geocoding'),
```

Comment:
Why is the name "geocoding" mnemonic for a function that fetches the user's physical address from GPS data?

Reply:
going from lat,lng to address is actually reverse geocoding, I'll think of another name. 
What about `MapsReverseGeocodingView` ? Or `UserLocationView` ? I'm open to suggestions here

Reply:
Went with `ReverseGeocodingView`

Reply:
Is that clear than "address_lookup_from_geo"?

Reply:
> Is that clear than "address_lookup_from_geo"?

PEP8 tells us we should use CamelCase for class names.
AddressLookupFromGeoView isn't, to my mind, clearer than ReverseGeocodingView. 

Reverse Geocoding describes the process to go from lat,lng pair to a fully worded address. This is what this view does, I even omitted the Maps part because it's irrelevant, no matter the service we use, inputs and outputs are the same nature.

Address_lookup_from_geo otoh, is a bit confusing to me : address lookup ? What kind of address ? IP ? and geo ? What's geo ? Has to do with Geography but other than that ..